packages/admin/src/components/notification-banner/DownloadMessage.tsx

```diff
@@ -0,0 +1,36 @@
+type DownloadMessageProps = {
+  count: number;
+};
+
+const DownloadMessage = ({ count }: DownloadMessageProps) => {
+  return (
+    <div
+      className="govuk-notification-banner"
+      role="region"
+      aria-labelledby="govuk-notification-banner-title"
+      data-module="govuk-notification-banner"
+      data-testid="spotlight-banner"
```

Comment:
this should be removed if not used or renamed to something appropriate

Reply:
Removed this file now